### DISCUSSION TITLE
Workflow Debug: I'm getting an error of "Dimension 1" despite image being divisible by 64. Image and Workflow included.

### DISCUSSION THREAD

**Discussion by aegis72** [comfyanonymous/ComfyUI#1524] [Q&A] [answered]
This has a load of external plugins (nested nodes, RGThree, SimpleMath, WAS, and others) and is laid out for good groupings of like elements, not clean noodles. That said, I cleaned up the problem nodes in the bottom right with a clean left-to-right flow. If I can get past "anime lineart"  processor, then the whole shebang works great.

See Notations for explanation
![workflow-notated](https://github.com/comfyanonymous/ComfyUI/assets/118572301/1c65378d-816e-47e0-88ef-4679152d0d16)

I've been trying to figure this out all day. If anyone can ID where I am being very dumb, I'd be grateful.

**Comment by aegis72** [comment]
I've attached my drawing here, as well as two more I've made that fail.
Works:
![shehulk](https://github.com/comfyanonymous/ComfyUI/assets/118572301/2f1e7d5f-a414-4cb3-a4ea-25a2e10dd8c6)

Fails:
![adventurer_non_16_divisible_test_clipdrop-background-removal](https://github.com/comfyanonymous/ComfyUI/assets/118572301/6654395f-46c5-4f51-9abd-5090caae51b6)

![don-engel-mooring-final-greebled](https://github.com/comfyanonymous/ComfyUI/assets/118572301/6df8d0f2-dc8f-48bf-a365-ba24c66c7c7a)

I know I could probably just make them all conforming when I draw them, but I am trying to avoid that manual part of the process. Especially at work I can't always dictate how the images come in.

**Comment by okolenmi** [comment, marked as answer]
![изображение](https://github.com/comfyanonymous/ComfyUI/assets/55319814/3800b03b-e741-4755-b90a-2aa63c114b5e)
It's a custom node, so it would be better to report error to developer of this custom node.

https://github.com/Fannovel16/comfyui_controlnet_aux - I found it for you

**Reply by aegis72** [reply]
I figured the issue might be in my math or something more fundamental than a bug. I always assume that *I am the problem first :). But yes, I suppose it might be a bug so I'll ask over there as well. Thanks.

**Reply by okolenmi** [reply]
Developer can fix it even for bad image resolutions, anyway. The question is about he is motivated to do it or not.

**Reply by aegis72** [reply]
It appears he is! Continuing this over there....https://github.com/Fannovel16/comfyui_controlnet_aux/issues/47

**Reply by aegis72** [reply]
Fixed; updating the node seems to have cleared the bug and eliminated my extra math needs, cleaning up the workflow.
